contrib/freshmaker-cli

```diff
@@ -225,6 +248,49 @@ def get_freshmaker_events_by_ids(api_url, ids):
     return events
 
 
+def get_advisories_from_errata(errata_url, date_from=date.today() - timedelta(7),
+                               date_to=date.today()):
+    """
+    Sends the GET request to the Errata filter for the latest RHSA, in SHIPPED_LIVE state. Returns
+    the list of advisories that are IMPORTANT or CRITICAL and the issue date is between specified
+    dates.
+    """
+    if (date_to - date_from).days > 7:
+        print("The interval between dates should not exceed 7 days")
+        sys.exit(1)
+
+    krb_auth = HTTPKerberosAuth()
+    r = requests.get(errata_url, auth=krb_auth, timeout=30)
+    r.raise_for_status()
+    data = r.json()
+    return [
+        advisory["id"]
+        for advisory in data
+        if advisory["security_impact"] in ("Important", "Critical") and
+           date_from <= datetime.strptime(advisory["timestamps"]["issue_date"],
+                                          "%Y-%m-%dT%H:%M:%SZ").date() <= date_to
+    ]
+
+
+def get_errata_advisories_ignored_by_freshmaker(api_url, search_keys):
+    print("Getting Freshmaker events")
+    events = []
+    for search_key in search_keys:
```

Comment:
Optional: rename `search_key` and `search_keys` to `advisory` and `advisories`. I personally hate this `search_key` :D and I think it creates a lot of confusion in new developers, so maybe we should try using it as less as possible in the future. But this is optional.